go/vt/vtgate/semantics/table_collector.go

```diff
@@ -104,12 +109,98 @@ func (etc *earlyTableCollector) handleTableName(tbl sqlparser.TableName, aet *sq
 	etc.Tables = append(etc.Tables, tableInfo)
 }
 
+func (tc *tableCollector) visitRowAlias(ins *sqlparser.Insert, rowAlias *sqlparser.RowAlias) error {
```

Comment:
nitpick: this method could do with a bit of refactoring love to make it easier to read. maybe a couple of `extract method` or something would be enough